forge-core/src/main/java/forge/CardStorageReader.java

```diff
@@ -382,25 +382,14 @@ private List<String> readScript(final InputStream inputStream) {
      * @return a new Card instance
      */
     protected final CardRules loadCard(final CardRules.Reader reader, final File file) {
-        FileInputStream fileInputStream = null;
-        try {
-            fileInputStream = new FileInputStream(file);
+        try (InputStream fileInputStream = java.nio.file.Files.newInputStream(file.toPath())) {
```

Comment:
use `import java.nio.file.Files` ?

Reply:
Clashes with `import com.google.common.io.Files`, which is used for `Files.getNameWithoutExtension`.

Reply:
Sadly Java doesn't gave import aliases